### DISCUSSION TITLE
Vaillant EcoTec Plus

### DISCUSSION THREAD

**Discussion by rom-burner** [john30/ebusd#833] [General]
Been all over the wiki and forums and have registered for the next batch of adapters when available, but in the meantime I have two very basic questions:

What is 'bai', and how do I know if my boiler is supported in the configuration files?!

What I want to do is log data from, not control, my Vaillant Ecotec Plus boiler, specifically FlowT, ReturnT, Pump Speed, Modulation, Burner state, some measure of power output/gas flow (and I'm thinking about adding a temp sensor in a pocket in the hot water cylinder to register that temp into the boiler).  I have no controller attached to the boiler, it simply receives on/off heat demand commands from the independent room/floor/hot water timers/stats and the boiler modulates itself.

My main question is, would it be possible to power an eBus Adapter 3.1(wifi) with a standard USB wall transformer and connect it to the empty eBus connectors on the boiler board that doesn't have a controller attached and have it access this data via an eBusd instance (that i'll run in a docker container on a server on my LAN and poll with Telegraf/InfluxDB)?  Or do I have to use the eBus Adapter as a pseudo-controller to stimulate the data chatter on which it could also listen-in?

Has anyone else achieved this simple use case and know what data is available?

Thanks

**Comment by nimmerpeople** [comment]
I'm not sure I understand your question, but have a look at this site. I set this up with my Vaillant Ecotec Plus.  
https://fromeijn.nl/

My adapter(not wifi) lights up when connected to the boiler terminals, without any external USB. but that's prob better understood by those who understand the physical side.

I use a super simple Pi to run Home assistant which has all the native stuff needed - inc ebusd.

Hope this helps a bit.

**Reply by Dan-McG** [reply]
thanks @nimmerpeople that's quite helpful! my main question was: can an ebus adapter listen to useful data from the boiler without a vaillant controller stimulating the creation of that data, and although i don't know your particular boiler, your project seems to say 'yes'.

my second question is what data is available, and i see from your project at least water pressure is there, so am optimistic flow and return temperatures etc will also be available.

i was also curious whether the boiler's ebus terminals could power the boards available, and i see that's how you power yours, but i suspect the wifi boards may need a separate power source via the usb header.

but i still don't know what a 'bai' is!

thanks for your help

**Reply by nimmerpeople** [reply]
PS I have the same boiler as you.. there's LOADS of info via google on what Vaillant info is available, the example i gave you is Vaillant too..

**Comment by nimmerpeople** [comment]
Yes, I don't have a vaillant controller and one isn't required - the link I
sent is an example of how people are building their own controller in
software using the ebus controller.

The data I believe will depend on the boiler - but yes, flow and return
temps are available in mine and every other example I've seen.

and re power - I think you're wrong - the ebus wifi adapters I've seen - eg
https://www.lectronz.com/products/ebus-to-wifi-adapter-module - take power
from the ebus terminals alone.


Happy days for you.

PS I have never managed to find out what bai means yet either!
